### ISSUE TITLE
a11y Contrast issues on pagination

### ISSUE THREAD

**Issue by mberkowski** [UMNLibraries/cdm-blacklightify#105]
Pagination links Previous/Next/... need better contrast. These come out of blacklight CSS
![image](https://github.com/user-attachments/assets/d4edd3f2-3827-4d54-aeca-97e905d16d0a)

**Comment by sourslaw** [comment]
Closing. Can reopen after updated theming and still issue remains
